extended/src/main/java/io/kubernetes/client/extended/kubectl/KubectlRollout.java

```diff
@@ -308,4 +330,241 @@ public V1PodTemplateSpec execute() throws KubectlException {
       }
     }
   }
+
+  class KubectlRolloutUndo extends Kubectl.ResourceBuilder<ApiType, KubectlRolloutUndo>
```

Comment:
`static`

Reply:
We need to access the apiTypeClass of the external class,so class KubectlRolloutUndo can't be static.